src/corelibs/U2Core/src/util/U2SequenceUtils.h

```diff
@@ -135,6 +137,8 @@ class U2CORE_EXPORT U2SequenceImporter {
 
     const DNATranslation* aminoTT = nullptr;
     const DNATranslation* complTT = nullptr;
+
+    std::bitset<256> alphabetCharacterHit;
```

Comment:
nit: this is not a blocker, but why 256 but not CHAR_MAX (127) ?

Reply:
added conversion to uchar